product_attribute_model_link/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Product Attribute Model Link",
     "summary": "Use any model records as product attribute values",
-    "version": "16.0.1.0.1",
+    "version": "16.0.1.0.2",
```

Comment:
This is considered a significant change. So need to increment the `x` version component. Here is a reference in the  guidelines: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers